.gitmodules

```diff
@@ -0,0 +1,24 @@
+[submodule "dac-ansible/roles/andrewrothstein.pki"]
```

Comment:
We have the dependencies here:
https://github.com/RSE-Cambridge/data-acc/blob/master/dac-ansible/requirements.yml